File: app/src/main/java/com/example/ecnill/separatedlogin/utils/SnackbarUtils.java
package com.example.ecnill.separatedlogin.utils;

import android.graphics.Color;
import android.support.design.widget.Snackbar;
import android.view.View;
import android.widget.TextView;

import butterknife.ButterKnife;

/**
 * Created by ecnill on 5/9/2017.
 */

public abstract class SnackbarUtils {

    public static Snackbar createSnackbar(View view, String message) {
        Snackbar snackbar = Snackbar.make(view, message, Snackbar.LENGTH_LONG);
        snackbar.setActionTextColor(Color.RED);
        View snackbarView = snackbar.getView();
        snackbarView.setBackgroundColor(Color.WHITE);

        TextView textView = ButterKnife.findById(snackbarView, android.support.design.R.id.snackbar_text);
        textView.setTextColor(Color.BLUE);
        return snackbar;
    }

}

File: app/src/main/java/com/example/ecnill/separatedlogin/utils/SnackbarUtils.kt
package com.example.ecnill.separatedlogin.utils

import android.graphics.Color
import android.support.design.widget.Snackbar
import android.view.View
import android.widget.TextView

import butterknife.ButterKnife

/**
 * Created by ecnill on 5/9/2017.
 */

object SnackbarUtils {

    fun createSnackbar(view: View, message: String): Snackbar {
        val snackbar = Snackbar.make(view, message, Snackbar.LENGTH_LONG)
        snackbar.setActionTextColor(Color.RED)
        snackbar.view.setBackgroundColor(Color.WHITE)
        ButterKnife.findById<TextView>(snackbar.view, android.support.design.R.id.snackbar_text).setTextColor(Color.BLUE)
        return snackbar
    }

}
